frameworks/PHP/mixphp/views/fortunes.php

```diff
@@ -0,0 +1,4 @@
+<!DOCTYPE html><html><head><title>Fortunes</title></head><body><table><tr><th>id</th><th>message</th></tr>
+<?php foreach ($rows as $row) : ?>
+<tr><td><?= $row['id'] ?></td><td><?= htmlspecialchars($row['message']) ?></td></tr>
```

Comment:
An ORM map rows to objects, you have classified the ORM as Mico.
But here you are using an array no objects!!

Reply:
Modify the https://github.com/TechEmpower/FrameworkBenchmarks/pull/6724/commits/4d5d3e405123708fa069fad1229bbd29253e33af

Reply:
@judofyr Pecl install made an error during build Docker. It may be a network problem. Please kindly approve it again